projects/hermes/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: "https://github.com/facebook/hermes"
+language: C++
+primary_contact: "neildhar@fb.com"
+auto_ccs:
+  - "dulinr@fb.com"
+vendor_ccs:
+  - "oss-fuzz@fb.com"
+fuzzing_engines:
+  - libfuzzer
+  - afl
+sanitizers:
+  - address
+  - undefined
```

Comment:
Can we add `thread` in here as well? Shortly TSAN will become more important to Hermes

Reply:
I don't believe OSS-Fuzz supports TSAN